examples/llm_plus_gnn/nvtx_run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Check if the user provided a Python file
+if [ -z "$1" ]; then
+    echo "Usage: $0 <python_file>"
```

Comment:
try to group nvtx_examples and nvtx_run nd other profiling under one profiling folder

Reply:
Done